templates/index.html

```diff
@@ -3,15 +3,66 @@
 
 {% block content %}
 
+<style>

```

Comment:
[nit] can i ask why css styles are defined in html file? not in 'css' directory?

Reply:
For component-specific scoped styles like these, I believe co-locating them with the component HTML is cleaner.